composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/form": "^2.3.5 || ^3.0",
         "symfony/http-foundation": "^2.3 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
+        "symfony/security": "^2.3 || ^3.0",
```

Comment:
It has to do with this line : https://github.com/sonata-project/SonataCoreBundle/pull/364/files#diff-ab34013c1c5e23c243a5c45874748062R52